src/utils/substitute-svgs.js

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs';
+import path, { extname } from 'path';
+
+const selfClosingSvgs = /<svg([^>]*)\/>/gi;
+const selfClosingSvgsLocal = /<svg([^>]*)\/>/gi;
```

Comment:
Are these 2 regular expressions intended to be exactly the same?

Reply:
Nope